notifications/src/main/java/com/dojo/notifications/model/leaderboard/Leaderboard.java

```diff
@@ -8,18 +8,17 @@
 import com.hubspot.slack.client.models.blocks.objects.Text;
 import com.hubspot.slack.client.models.blocks.objects.TextType;
 
-import java.util.Collections;
-import java.util.List;
 import java.util.Objects;
+import java.util.TreeSet;
 import java.util.concurrent.atomic.AtomicInteger;
 
 public class Leaderboard {
 
     private static final UserDetails COMMON = null;
 
-    private final List<Participant> participants;
+    private final TreeSet<Participant> participants;
```

Comment:
We usually specify the type as interface - Set (like it was previously with List)
We would use TreeSet only if we were using a method from TreeSet class